homeassistant/components/velux/__init__.py

```diff
@@ -0,0 +1,72 @@
+"""
```

Comment:
Are you planning to add more Velux-related platform or features? If not, then it doesn't make sense to put it in a separate directory.